eng/Build.ps1

```diff
@@ -18,7 +18,7 @@
 param (
     [string][Alias('c')]$configuration = "Debug",
     [string][Alias('v')]$verbosity = "m",
-    [string]$msbuildEngine = "vs",
+    [string]$msbuildEngine = "dotnet",
```

Comment:
This change won't pull in the `%VS160COMNTOOLS%` or `%VSAPPDIR%` variables so the VS tests will fail.  Your changes to `eng/common/*` will be overwritten by arcade.

Reply:
@brettfo, yes I will get them another.  We have to use dotnet to build, because otherwise the resources are compiled incorrectly.